
<launch>
	<arg name="compressed" default="true"/>
	<arg name="stereo" default="true"/>
	<arg name="point_cloud_topic" default="/rtabmap/scan_map"/>
	<node pkg="wallie_one" type="zed_sub_stereo" name="zed_stereo_node" output="screen" >
		

	<node pkg="wallie_one" type="pcl_reader_no_imagew" name="pcl_reader_no_imagew" output="screen" launch-prefix="xterm -e gdb --args" >

   </node> 
	<node pkg="wallie_one" type="pcl_buffer" name="pcl_buffer_node" output="screen" >

		<param name="point_cloud_topic" type="string" value="$(arg point_cloud_topic)" />

	   </node> 
   	<node pkg="rqt_reconfigure" type="rqt_reconfigure" name="rqt_reconfigure_node" output="screen" >

   </node> 
 	<node pkg="wallie_one" type="pcl_transformer.py" name="blank_image_pub" output="screen">


   </node> 
 	<node pkg="wallie_one" type="TFBroadcaster.py" name="hmd_transform" output="screen">


   </node>
 	<node pkg="joy" type="joy_node" name="joy_node" output="screen">
		<arg name="autorepeat_rate" value="5.0" />

   </node>

	<remap from="/pc_roi" to="/filtered_cloud"/>
   </node> 
	<include file="$(find wallie_one)/launch/move_base.launch">
	</include>

	
</launch>
